debian/changelog

```diff
@@ -1,3 +1,15 @@
+eos-meta (1.5.1) UNRELEASED; urgency=medium
```

Comment:
We do not touch the changelog -- please put these notes in the commit message instead.